app/models/laboratory.rb

```diff
@@ -30,7 +30,10 @@ class Laboratory < ApplicationRecord
     **LAB_TYPE_TO_CODE
   }.invert.freeze
 
-  validates :result, inclusion: { in: ['positive', 'negative', 'indeterminate', 'other', nil, ''] }
+  validates :result, inclusion: {
+    in: [*RESULT_TO_CODE.keys, nil, ''],
+    message: "is not an acceptablle value, acceptable values are: '#{RESULT_TO_CODE.keys.reject(&:blank?).join("', '")}'"
```

Comment:
`acceptablle` typo needs to be fixed.

Also when repopulating the DB with `DAYS=5 bundle exec rails db:drop db:create db:schema:load admin:import_or_update_jurisdictions demo:setup demo:populate` I get this error.

![Screen Shot 2021-07-08 at 1 34 33 PM](https://user-images.githubusercontent.com/24628687/124980133-3db02b00-dff1-11eb-9f3d-474381669888.png)

Reply:
I think it may need to be:

```ruby
  validates :result, inclusion: {
    in: [*RESULT_TO_CODE.keys.map(&:to_s), nil, ''],
    message: "is not an acceptablle value, acceptable values are: '#{RESULT_TO_CODE.keys.reject(&:blank?).join("', '")}'"
  }
```

because RESULT_TO_CODE.keys returns an array of symbols, not strings.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/991/files#diff-7d2113c793528ea2c21a9bbc24918900a3ee6fd1bd396956de26304a9870162eR34